docs/csharp/language-reference/compiler-options/target-compiler-option.md

```diff
@@ -31,7 +31,7 @@ The **-target** compiler option can be specified in one of four forms:
  [-target:winmdobj](./target-winmdobj-compiler-option.md)  
  To create an intermediate .winmdobj file.  
   
- Unless you specify **-target:module**, **-target** causes a .NET Framework assembly manifest to be placed in an output file. For more information, see [Assemblies in .NET](../../../standard/assembly/index.md) and [Common Attributes](../../programming-guide/concepts/attributes/common-attributes.md).  
+ Unless you specify **-target:module**, **-target** causes a .NET Framework assembly manifest to be placed in an output file. For more information, see [Assemblies in .NET](../../../standard/assembly/index.md) and [Common Attributes](../attributes/global.md).  
```

Comment:
Thanks for catching this. I thought I had found all these links when I updated the related articles.